pkg/sql/pgwire/command_result.go

```diff
@@ -345,6 +351,27 @@ func (c *conn) newMiscResult(pos sql.CmdPos, typ completionMsgType) *commandResu
 	return r
 }
 
+// portalSuspensionStrategy determines how the suspension of the portal should
+// be handled by limitedCommandResult. It mirrors relevant enum values from PG
+// (https://github.com/postgres/postgres/blob/2f9661311b83dc481fc19f6e3bda015392010a40/src/include/utils/portal.h#L89)
+// and is currently used only to populate the command tag with the correct
+// number of "affected rows."
+type portalSuspensionStrategy int
+
+const (
+	// selectPortalSuspensionStrategy is a portalSuspensionStrategy for a
+	// portal that consists of a SELECT statement. The execution of such
+	// statements can be "truly" suspended, so PG emits the actual number of
+	// returned rows since the previous suspension (if there was one).
+	selectPortalSuspensionStrategy portalSuspensionStrategy = iota
```

Comment:
These comments should document how, or if, they relate to which enum values in postgres.